docs/faas/faas-metrics.md

```diff
@@ -54,9 +54,9 @@ of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10
 <!-- markdownlint-capture -->
 <!-- markdownlint-disable -->
 
-| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
-| -------- | --------------- | ----------- | -------------- | --------- |
-| `faas.invoke_duration` | Histogram | `s` | Measures the duration of the function's logic execution | ![Development](https://img.shields.io/badge/-development-blue) |
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability | Entity Associations |
+| -------- | --------------- | ----------- | -------------- | --------- | ------ |
+| `faas.invoke_duration` | Histogram | `s` | Measures the duration of the function's logic execution | ![Development](https://img.shields.io/badge/-development-blue) |  |
```

Comment:
these should probably be associated with faas resource

Reply:
Yes, but see the PR description. THis only updates semconv that are trying to stabilize soon.